snuba/datasets/plans/storage_plan_builder.py

```diff
@@ -176,6 +178,23 @@ def build_and_rank_plans(
         mappers = storage_connection.translation_mappers
         cluster = self.get_cluster(storage, query, settings)
 
+        # Return failure if storage readiness state is not supported in current environment
+        assert isinstance(storage, ReadableTableStorage)
```

Comment:
Why this assert?

Reply:
Moved to base class. But `assert` is still required to access to storage_key.